container/health.go

```diff
@@ -19,12 +19,7 @@ func (s *Health) String() string {
 		return types.Unhealthy
 	}
 
-	switch s.Status {
-	case types.Starting:
-		return "health: starting"
```

Comment:
It makes sense, just want to make sure we aren't changing a user-facing value or if we are we know the ramifications.